frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -395,7 +398,81 @@
                     vm.submissionResult = response;
                     // navigate to challenge page
                     // $state.go('web.challenge-page.overview');
-                    vm.stopLoader();
+                    if (vm.submissionResult.next === null) {
+                        vm.isNext = 'disabled';
+                    } else {
+                        vm.isNext = '';
+
+                    }
+                    if (vm.submissionResult.previous === null) {
+                        vm.isPrev = 'disabled';
+                    } else {
+                        vm.isPrev = '';
+                    }
+                    if (vm.submissionResult.next !== null) {
+                        vm.currentPage = vm.submissionResult.next.split('page=')[1] - 1;
+                    } else {
+                        vm.currentPage = 1;
+                    }
+
+                    vm.load = function(url) {
+                        // loader for exisiting teams
+                        vm.isExistLoader = true;
+                        vm.loaderTitle = '';
+                        vm.loginContainer = angular.element('.exist-team-card');
+
+                        // show loader
+                        vm.startLoader = function(msg) {
+                            vm.isExistLoader = true;
+                            vm.loaderTitle = msg;
+                            vm.loginContainer.addClass('low-screen');
+                        };
+
+                        // stop loader
+                        vm.stopLoader = function() {
+                            vm.isExistLoader = false;
+                            vm.loaderTitle = '';
+                            vm.loginContainer.removeClass('low-screen');
+                        };
+
+                        vm.startLoader("Loading Teams");
+                        if (url !== null) {
+
+                            //store the header data in a variable
+                            var headers = {
+                                'Authorization': "Token " + userKey
+                            };
+
+                            //Add headers with in your request
+                            $http.get(url, { headers: headers }).then(function(response) {
+                                // reinitialized data
+                                var status = response.status;
+                                var response = response.data;
+                                vm.submissionResult = response;
+
+                                // condition for pagination
+                                if (vm.submissionResult.next === null) {
+                                    vm.isNext = 'disabled';
+                                    vm.currentPage = vm.submissionResult.count / 3;
```

Comment:
Edit: I forgot to change back to 10 as I was testing with 3. my bad . Changing it.